app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -700,12 +701,22 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
         routeModeView.route = null
     }
 
+    private fun setBoundingBox() {
+        val menuItem = optionsMenu?.findItem(R.id.action_search)
+        var mdisp : Display = getWindowManager().getDefaultDisplay();
+        var minLatLon = mapController?.coordinatesAtScreenPosition(mdisp.width.toDouble(), mdisp.height.toDouble())
+        var maxLatLon = mapController?.coordinatesAtScreenPosition(0.0, 0.0);
+        var bbox: BoundingBox = BoundingBox(minLatLon?.latitude as Double, minLatLon?.longitude as Double,
+                maxLatLon?.latitude as Double, maxLatLon?.longitude as Double)
```

Comment:
Probably makes sense to take the `min` and `max` of the two latitudes and longitudes when creating the bounding box, either here or within the `BoundingBox` constructor. If the map view is rotated, the left side of the screen may no longer be the West-most side.